mypy/util.py

```diff
@@ -519,6 +519,10 @@ def parse_gray_color(cup: bytes) -> str:
     return gray
 
 
+def should_force_color() -> bool:
+    return bool(int(os.getenv("MYPY_FORCE_COLOR", os.getenv("FORCE_COLOR", "0"))))
```

Comment:
fyi, if `FORCE_COLOR` is used for a different purpose (e.g. a different python library), it could contain a string, and this would raise an error.

Reply:
Yeah, would be better to just check whether `FORCE_COLOR` exists, with any value (even 0!).

Reply:
> (even 0!)

...Really? If somebody puts `FORCE_COLOR: 0`, they expect it to force color??

Reply:
> > (even 0!)
> 
> ...Really? If somebody puts `FORCE_COLOR: 0` in a GitHub Actions workflow, they expect it to force color??

From a review of the Python projects @hugovk linked to in https://github.com/python/mypy/issues/13806#issue-1395502590, it looks like the answer is: yup! `tox` is the only project on that list that will treat `FORCE_COLOR: 0` as indicating that the user does not want color.

Reply:
- #13853

Reply:
@AlexWaygood thanks, but this is not what i was talking about. i was talking about a possible `ValueError` in case the variable is not a number (e.g. `False`).

Reply:
> ...Really? If somebody puts `FORCE_COLOR: 0` in a GitHub Actions workflow, they expect it to force color??

Yes, that's what `NO_COLOR` does! 😅 

> Command-line software which adds ANSI color to its output by default should check for a `NO_COLOR` environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

https://no-color.org/

Here's how pytest does it:

https://github.com/pytest-dev/pytest/blob/b7d4de1ea983d3c4d3d9e235f63942abc6bfb167/src/_pytest/_io/terminalwriter.py#L26-L37